workshop/exercise-5/README.md

```diff
@@ -85,10 +87,29 @@ The IBM Ingress service provides IBM Cloud users with a secure, reliable, and sc
     ```
     Output:
     ```shell
-    Ingress subdomain:	guestbook-242887.us-east.containers.mybluemix.net
+    Ingress subdomain:	mycluster.us-east.containers.mybluemix.net
     ```
 
-2. Add the subdomain that you retrieved in the previous step as `host` in the `guestbook-frontdoor.yaml` file, which you can find in the `istio101/workshop/plans` directory.
+2. Prepend `guestbook.` to the subdomain that you retrieved in the previous step. This new url will serve as `host` in the `guestbook-frontdoor.yaml` file, which you can find and edit in the `istio101/workshop/plans` directory.
+
+    The file should now look something like this:
+
+    ```yaml
+    apiVersion: extensions/v1beta1
+    kind: Ingress
+    metadata:
+      name: guestbook-ingress
+      namespace: istio-system
```

Comment:
does the guestbook-ingress have to be placed in istio-system namespace?  If not, I'd prefer not to as it is a sample app.

Reply:
because it's forwarding to the istio-ingressgateway it needs to be in the same namespace.

Reply:
Thanks, I compared with my old yaml, yes, it does need to be in that namespace.